projects/make-a-vis/src/themes/shared/common-colors.scss

```diff
@@ -6,6 +6,8 @@ $teal-400: rgba(38, 166, 154, 1);
 $deeporange-900: rgba(191, 54, 12, 1);
 $red-a700: rgba(213, 0, 0, 1);
 $yellow-a200: rgba(255, 255, 0, 1);
+$yellow-a200-20palpha: rgba(255, 255, 0, 0.2);
+$yellow-a200-50palpha: rgba(255, 255, 0, 0.5);
```

Comment:
Please use same formatting for percent alpha as the rest of the ones with that feature (see 2 rows down).